samples/multiplatform/src/jsTest/mocha/src/test/kotlin/ch/tutteli/SmokeTest.kt

```diff
@@ -0,0 +1,65 @@
+package ch.tutteli
+
+import ch.tutteli.atrium.api.fluent.en_GB-js.*
+import ch.tutteli.atrium.api.verbs.assertThat
+import ch.tutteli.atrium.assertions.Assertion
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.domain.builders.AssertImpl
+import ch.tutteli.atrium.reporting.RawString
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+import ch.tutteli.atrium.translations.DescriptionBasic
+import kotlin.test.Test
+
+class SmokeTest {
+    @Test
+    fun toBe_canBeUsed() {
+        assertThat(1).isLessThan(2)
```

Comment:
change to 
```suggestion
        assertThat(1).toBe(2) 
```
so that you see the test fails in the build